src/ucs/datastruct/bitmap.h

```diff
@@ -536,7 +536,7 @@ ucs_bitmap_ffs(const ucs_bitmap_word_t *bitmap_words, size_t num_words,
             return _UCS_BITMAP_BIT_INDEX(first_bit_in_word, word_index);
         }
 
-        mask = UINT64_MAX;
+        mask = ((uint64_t) -1);
```

Comment:
UCS_MASK_SAFE(UCS_BITMAP_BITS_IN_WORD)

Reply:
@yosefe I got below comple error when I change to this:
```In file included from /root/bilei/ucx/contrib/../src/ucs/memory/memory_type.h
                 from /root/bilei/ucx/contrib/../test/gtest/common/mem_buffer
                 from /root/bilei/ucx/contrib/../test/gtest/common/mem_buffer
/root/bilei/ucx/contrib/../src/ucs/datastruct/bitmap.h: In function 'int ucs_t*, size_t, size_t)':
/root/bilei/ucx/contrib/../src/ucs/sys/compiler_def.h:74:44: error: left shif]
 #define UCS_BIT(i)               (1ul << (i))
                                            ^
/root/bilei/ucx/contrib/../src/ucs/sys/compiler_def.h:77:35: note: in expansi
 #define UCS_MASK(i)              (UCS_BIT(i) - 1)
                                   ^
/root/bilei/ucx/contrib/../src/ucs/sys/math.h:93:40: note: in expansion of ma
     (((_i) >= 64) ? ((uint64_t)(-1)) : UCS_MASK(_i))
                                        ^
/root/bilei/ucx/contrib/../src/ucs/datastruct/bitmap.h:540:16: note: in expan
         mask = UCS_MASK_SAFE(UCS_BITMAP_BITS_IN_WORD);```

Simplifying the error is caused by the code "1ul << 64".

Reply:
Bypassed the error by changing to:
···
        size_t bits;
        bits = UCS_BITMAP_BITS_IN_WORD;
        mask = UCS_MASK_SAFE(bits);
···